[SQL] Add Ice Brand and Flametongue item_mods.sql, and a couple other item_mod corrections

helixhamin opened the pull request:
Also corrected a few mistakes in other items I found.

<!-- Remove space and place 'x' mark between square [] brackets or click the checkbox after saving to affirm the following points: -->
<!-- (it should look like this: - [x] I have ...) -->
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my PR will be ignored.
- [x] I understand I should leave resolving conversations to the LandSandBoat team so that reviewers won't miss what was said.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have _**tested my code and the things my code has changed**_ since the last commit in the PR and will test after any later commits.

## What does this pull request do?
Adds in the item modifiers for Ice Brand and Flametongue. I also corrected a couple things with some other items I found had issues.

## Steps to test these changes
As GM, !additem 20672 and 20673. Make sure stats show properly, and hit things with them to make sure the additional effects work properly.

helixhamin committed:
Add Ice Brand and Flametongue item_mods.sql
Also corrected a few mistakes in other items I found.

Xaver-DaRed reviewed (commented):
Looks good in general just a couple of things.
- Could you align the comments, per item?
- For non-existing modifiers, could you add a TODO in the comment?
    For example `-- TODO: unimplemented modifier + x%`

Xaver-DaRed commented on sql/item_mods.sql:
```diff
@@ -2109,7 +2109,7 @@ INSERT INTO `item_mods` VALUES (10615,384,300); -- HASTE_GEAR: 300
 -- Rheic Schuhs
 INSERT INTO `item_mods` VALUES (10616,1,26);      -- DEF: 26
-INSERT INTO `item_mods` VALUES (10616,161,-3000); -- DMGPHYS: -3000
+INSERT INTO `item_mods` VALUES (10616,161,-300); -- DMGPHYS: -3%
```
Nice catch

helixhamin commented:
> Looks good in general just a couple of things.
> 
> * Could you align the comments, per item?
> * For non-existing modifiers, could you add a TODO in the comment?
>   For example `-- TODO: unimplemented modifier + x%`

Aligned per request. Changed Flametongue missing mod note to:
`-- TODO: Physical damage limit +5%`

Removed the sql text from it to minimize the amount of text and show this is just something needs to be implemented.
